azure/azure-eventhub-consumer/pom.xml

```diff
@@ -0,0 +1,67 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+   xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+   <modelVersion>4.0.0</modelVersion>
+   <parent>
+      <groupId>org.springframework.boot</groupId>
+      <artifactId>spring-boot-starter-parent</artifactId>
+      <version>2.1.3.RELEASE</version>
+   </parent>
+   <groupId>org.eclipse.ditto.examples.azure</groupId>
+   <artifactId>azure-eventhub-consumer</artifactId>
+   <version>0.0.1-SNAPSHOT</version>
+   <name>azure-eventhub-consumer</name>
+   <description>Example application that consumes Events from Eclipse Ditto by Azure Event Hub.</description>
+
+   <properties>
+      <java.version>11</java.version>
```

Comment:
@kaizimmerm I can confirm that the example also works Java 8.

Reply:
done